test/passport-saml/capturedSamlRequests.spec.ts

```diff
@@ -859,6 +859,73 @@ const capturedSamlRequestChecks: SamlCheck[] = [
       },
     },
   },
+  {
```

Comment:
Nice clean addition of another test leveraging the framework. Thanks!

Reply:
Thank you :)